addons/transparent-orphans/addon.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Transparent orphan blocks",
+  "description": "Makes any blocks that are orphaned (unused with no parent block) transparent.",
+  "tags": ["editor"],
```

Comment:
`beta` tag maybe? (we often do that on new addons)

Reply:
Yeah, I thought about that